### ISSUE TITLE
Experiment Help

### ISSUE THREAD

**Issue by KaneLi01** [sunyx523/StEik#1]
Hello author, I am unable to reproduce the results of the paper on certain datasets. Can you provide me with some details and assistance? Thank you.

**Comment by hyperzy** [comment]
Thanks for your interest in our work. Can you elaborate more on the issues you encountered? E.g. hardwares, environments, any specific modifications made.

**Comment by KaneLi01** [comment]
I trained your model on GPU RTX 3090 Ti, used virtual environment created by anaconda and ran the bash file with default parameter configuration. This is my conda list:
Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main
_openmp_mutex             5.1                       1_gnu
absl-py                   2.1.0                    pypi_0    pypi
addict                    2.4.0                    pypi_0    pypi
anyio                     4.3.0                    pypi_0    pypi
apptools                  5.2.1                    pypi_0    pypi
argon2-cffi               23.1.0                   pypi_0    pypi
argon2-cffi-bindings      21.2.0                   pypi_0    pypi
arrow                     1.3.0                    pypi_0    pypi
asttokens                 2.4.1                    pypi_0    pypi
astunparse                1.6.3                    pypi_0    pypi
async-lru                 2.0.4                    pypi_0    pypi
attrs                     23.2.0                   pypi_0    pypi
babel                     2.14.0                   pypi_0    pypi
beautifulsoup4            4.12.3                   pypi_0    pypi
bleach                    6.1.0                    pypi_0    pypi
blinker                   1.7.0                    pypi_0    pypi
ca-certificates           2023.12.12           h06a4308_0
certifi                   2024.2.2                 pypi_0    pypi
cffi                      1.16.0                   pypi_0    pypi
charset-normalizer        3.3.2                    pypi_0    pypi
clarabel                  0.9.0                    pypi_0    pypi
click                     8.1.7                    pypi_0    pypi
comm                      0.2.2                    pypi_0    pypi
configargparse            1.7                      pypi_0    pypi
configobj                 5.0.8                    pypi_0    pypi
cvxpy                     1.5.1                    pypi_0    pypi
cycler                    0.12.1                   pypi_0    pypi
dash                      2.16.1                   pypi_0    pypi
dash-core-components      2.0.0                    pypi_0    pypi
dash-html-components      2.0.0                    pypi_0    pypi
dash-table                5.0.0                    pypi_0    pypi
debugpy                   1.8.1                    pypi_0    pypi
decorator                 5.1.1                    pypi_0    pypi
defusedxml                0.7.1                    pypi_0    pypi
ecos                      2.0.13                   pypi_0    pypi
envisage                  7.0.3                    pypi_0    pypi
exceptiongroup            1.2.0                    pypi_0    pypi
executing                 2.0.1                    pypi_0    pypi
fastjsonschema            2.19.1                   pypi_0    pypi
flask                     3.0.2                    pypi_0    pypi
flatbuffers               24.3.25                  pypi_0    pypi
fqdn                      1.5.1                    pypi_0    pypi
gast                      0.6.0                    pypi_0    pypi
google-pasta              0.2.0                    pypi_0    pypi
grpcio                    1.62.1                   pypi_0    pypi
h11                       0.14.0                   pypi_0    pypi
h5py                      3.11.0                   pypi_0    pypi
httpcore                  1.0.4                    pypi_0    pypi
httpx                     0.27.0                   pypi_0    pypi
idna                      3.6                      pypi_0    pypi
imageio                   2.9.0                    pypi_0    pypi
importlib-metadata        7.0.2                    pypi_0    pypi
ipykernel                 6.29.3                   pypi_0    pypi
ipython                   8.18.1                   pypi_0    pypi
ipywidgets                8.1.2                    pypi_0    pypi
isoduration               20.11.0                  pypi_0    pypi
itsdangerous              2.1.2                    pypi_0    pypi
jedi                      0.19.1                   pypi_0    pypi
jinja2                    3.1.3                    pypi_0    pypi
joblib                    1.3.2                    pypi_0    pypi
json5                     0.9.24                   pypi_0    pypi
jsonpointer               2.4                      pypi_0    pypi
jsonschema                4.21.1                   pypi_0    pypi
jsonschema-specifications 2023.12.1                pypi_0    pypi
jupyter                   1.0.0                    pypi_0    pypi
jupyter-client            8.6.1                    pypi_0    pypi
jupyter-console           6.6.3                    pypi_0    pypi
jupyter-core              5.7.2                    pypi_0    pypi
jupyter-events            0.10.0                   pypi_0    pypi
jupyter-lsp               2.2.4                    pypi_0    pypi
jupyter-server            2.13.0                   pypi_0    pypi
jupyter-server-terminals  0.5.3                    pypi_0    pypi
jupyterlab                4.1.5                    pypi_0    pypi
jupyterlab-pygments       0.3.0                    pypi_0    pypi
jupyterlab-server         2.25.4                   pypi_0    pypi
jupyterlab-widgets        3.0.10                   pypi_0    pypi
kaleido                   0.2.1                    pypi_0    pypi
keras                     3.4.1                    pypi_0    pypi
kiwisolver                1.4.5                    pypi_0    pypi
ld_impl_linux-64          2.38                 h1181459_1
libclang                  18.1.1                   pypi_0    pypi
libffi                    3.4.4                h6a678d5_0
libgcc-ng                 11.2.0               h1234567_1
libgomp                   11.2.0               h1234567_1
libstdcxx-ng              11.2.0               h1234567_1
markdown                  3.6                      pypi_0    pypi
markdown-it-py            3.0.0                    pypi_0    pypi
markupsafe                2.1.5                    pypi_0    pypi
matplotlib                3.3.4                    pypi_0    pypi
matplotlib-inline         0.1.6                    pypi_0    pypi
mayavi                    4.8.2                    pypi_0    pypi
mdurl                     0.1.2                    pypi_0    pypi
mistune                   3.0.2                    pypi_0    pypi
ml-dtypes                 0.4.0                    pypi_0    pypi
namex                     0.0.8                    pypi_0    pypi
nbclient                  0.10.0                   pypi_0    pypi
nbconvert                 7.16.3                   pypi_0    pypi
nbformat                  5.10.3                   pypi_0    pypi
ncurses                   6.4                  h6a678d5_0
nest-asyncio              1.6.0                    pypi_0    pypi
networkx                  3.2.1                    pypi_0    pypi
notebook                  7.1.2                    pypi_0    pypi
notebook-shim             0.2.4                    pypi_0    pypi
numpy                     1.26.4                   pypi_0    pypi
open3d                    0.16.0                   pypi_0    pypi
opencv-python             4.10.0.82                pypi_0    pypi
openssl                   3.0.13               h7f8727e_0
opt-einsum                3.3.0                    pypi_0    pypi
optree                    0.12.1                   pypi_0    pypi
osqp                      0.6.7                    pypi_0    pypi
overrides                 7.7.0                    pypi_0    pypi
packaging                 24.0                     pypi_0    pypi
pandas                    2.2.1                    pypi_0    pypi
pandocfilters             1.5.1                    pypi_0    pypi
parso                     0.8.3                    pypi_0    pypi
pexpect                   4.9.0                    pypi_0    pypi
pillow                    8.2.0                    pypi_0    pypi
pip                       23.3.1           py39h06a4308_0
platformdirs              4.2.0                    pypi_0    pypi
plotly                    5.20.0                     py_0    plotly
plotly-orca               1.3.1                         1    plotly
plyfile                   0.7.2                    pypi_0    pypi
prometheus-client         0.20.0                   pypi_0    pypi
prompt-toolkit            3.0.43                   pypi_0    pypi
protobuf                  3.19.0                   pypi_0    pypi
psutil                    5.9.8                    pypi_0    pypi
ptyprocess                0.7.0                    pypi_0    pypi
pure-eval                 0.2.2                    pypi_0    pypi
pycparser                 2.21                     pypi_0    pypi
pyface                    8.0.0                    pypi_0    pypi
pygments                  2.17.2                   pypi_0    pypi
pyparsing                 3.1.2                    pypi_0    pypi
pyquaternion              0.9.9                    pypi_0    pypi
python                    3.9.18               h955ad1f_0
python-dateutil           2.9.0.post0              pypi_0    pypi
python-json-logger        2.0.7                    pypi_0    pypi
pytz                      2024.1                   pypi_0    pypi
pywavelets                1.5.0                    pypi_0    pypi
pyyaml                    6.0.1                    pypi_0    pypi
pyzmq                     25.1.2                   pypi_0    pypi
qdldl                     0.1.7.post2              pypi_0    pypi
qtconsole                 5.5.1                    pypi_0    pypi
qtpy                      2.4.1                    pypi_0    pypi
readline                  8.2                  h5eee18b_0
referencing               0.33.0                   pypi_0    pypi
requests                  2.31.0                   pypi_0    pypi
retrying                  1.3.4                    pypi_0    pypi
rfc3339-validator         0.1.4                    pypi_0    pypi
rfc3986-validator         0.1.1                    pypi_0    pypi
rich                      13.7.1                   pypi_0    pypi
rpds-py                   0.18.0                   pypi_0    pypi
scikit-image              0.18.1                   pypi_0    pypi
scikit-learn              1.4.1.post1              pypi_0    pypi
scipy                     1.6.2                    pypi_0    pypi
scs                       3.2.4.post2              pypi_0    pypi
send2trash                1.8.2                    pypi_0    pypi
setuptools                68.2.2           py39h06a4308_0
shapely                   2.0.3                    pypi_0    pypi
six                       1.16.0                   pypi_0    pypi
sniffio                   1.3.1                    pypi_0    pypi
soupsieve                 2.5                      pypi_0    pypi
sqlite                    3.41.2               h5eee18b_0
stack-data                0.6.3                    pypi_0    pypi
tenacity                  8.2.3                    pypi_0    pypi
tensorboard               2.17.0                   pypi_0    pypi
tensorboard-data-server   0.7.2                    pypi_0    pypi
tensorboardx              2.3                      pypi_0    pypi
tensorflow                2.17.0                   pypi_0    pypi
tensorflow-io-gcs-filesystem 0.37.1                   pypi_0    pypi
termcolor                 2.4.0                    pypi_0    pypi
terminado                 0.18.1                   pypi_0    pypi
threadpoolctl             3.3.0                    pypi_0    pypi
tifffile                  2024.2.12                pypi_0    pypi
tinycss2                  1.2.1                    pypi_0    pypi
tk                        8.6.12               h1ccaba5_0
tomli                     2.0.1                    pypi_0    pypi
torch                     1.12.0+cu113             pypi_0    pypi
torchaudio                0.12.0+cu113             pypi_0    pypi
torchvision               0.13.0+cu113             pypi_0    pypi
tornado                   6.4                      pypi_0    pypi
tqdm                      4.60.0                   pypi_0    pypi
traitlets                 5.14.2                   pypi_0    pypi
traits                    6.4.3                    pypi_0    pypi
traitsui                  8.0.0                    pypi_0    pypi
trimesh                   3.9.14                   pypi_0    pypi
types-python-dateutil     2.9.0.20240316           pypi_0    pypi
typing-extensions         4.10.0                   pypi_0    pypi
tzdata                    2024.1                   pypi_0    pypi
uri-template              1.3.0                    pypi_0    pypi
urllib3                   2.2.1                    pypi_0    pypi
vtk                       9.3.1                    pypi_0    pypi
wcwidth                   0.2.13                   pypi_0    pypi
webcolors                 1.13                     pypi_0    pypi
webencodings              0.5.1                    pypi_0    pypi
websocket-client          1.7.0                    pypi_0    pypi
werkzeug                  3.0.1                    pypi_0    pypi
wheel                     0.41.2           py39h06a4308_0
widgetsnbextension        4.0.10                   pypi_0    pypi
wrapt                     1.16.0                   pypi_0    pypi
xz                        5.4.6                h5eee18b_0
zipp                      3.18.0                   pypi_0    pypi
zlib                      1.2.13               h5eee18b_0

> 感谢您对我们的工作感兴趣。您能否详细解释一下您遇到的问题？例如，硬件、环境、所做的任何特定修改。



> 感谢您对我们的工作感兴趣。您能否详细解释一下您遇到的问题？例如，硬件、环境、所做的任何特定修改。

**Comment by KaneLi01** [comment]
> 感谢您对我们的工作感兴趣。您能否详细解释一下您遇到的问题？例如，硬件、环境、所做的任何特定修改。

I used the prompt of readme to train on the Shapenet dataset, and the final training result is as follows
![2024-10-10T09_46_30](https://github.com/user-attachments/assets/bb5bec53-9120-4dee-b68f-01e88cbd414a)
